pkg/acsengine/types.go

```diff
@@ -24,3 +24,36 @@ type V20160330ContainerService struct {
 	api.TypeMeta
 	*v20160330.ContainerService
 }
+
+//DCOSSpecConfig is the configurations of DCOS
+type DCOSSpecConfig struct {
+	DCOS173_BootstrapDownloadURL string
+	DCOS184_BootstrapDownloadURL string
+	DCOS187_BootstrapDownloadURL string
+}
+
+//DockerSpecConfig is default docker install script URL.
+type DockerSpecConfig struct {
+	DefaultDockerInstallScriptURL string
+}
+
+//KubernetesSpecConfig is the kubernetes container images used.
+type KubernetesSpecConfig struct {
+	DefaultKubernetesHyperkubeSpec         string
```

Comment:
@amanohar I agree with you that 10 extra configurable fields are too much. Based on your comments, I remove these extra fields and use base url to concat the whole image url. Thanks!